pkg/ctl/cmdutils/filter/nodegroup_filter.go

```diff
@@ -59,16 +66,16 @@ func (f *NodeGroupFilter) AppendIncludeNames(names ...string) {
 	f.delegate.AppendIncludeNames(names...)
 }
 
-// A stackLister lists nodegroup stacks
-type stackLister interface {
+// StackLister lists nodegroup stacks
+type StackLister interface {
 	ListNodeGroupStacks() ([]manager.NodeGroupStack, error)
 }
```

Comment:
Can we move this to the top of the file, have all definitions together?

Reply:
Yup, done!